cmd/ttn-lw-stack/commands/start.go

```diff
@@ -118,6 +119,20 @@ var startCommand = &cobra.Command{
 
 		var componentOptions []component.Option
 
+		cookieHashKey, cookieBlockKey := config.ServiceBase.HTTP.Cookie.HashKey, config.ServiceBase.HTTP.Cookie.BlockKey
+
+		if len(cookieHashKey) == 0 || isZeros(cookieHashKey) {
+			cookieHashKey = random.Bytes(64)
+			config.ServiceBase.HTTP.Cookie.HashKey = cookieHashKey
+			logger.WithField("hash_key", cookieHashKey).Warn("No cookie hash key configured, generated a random one")
+		}
+
+		if len(cookieBlockKey) == 0 || isZeros(cookieBlockKey) {
+			cookieBlockKey = random.Bytes(32)
+			config.ServiceBase.HTTP.Cookie.BlockKey = cookieBlockKey
+			logger.WithField("block_key", cookieBlockKey).Warn("No cookie block key configured, generated a random one")
```

Comment:
I wonder why we print secrets in the log. Is there a reason to add the actual value as field? It's quite hard for sysops to protect logs

Reply:
Good point. Removed.